megamek/src/megamek/client/ui/swing/boardview/StepSprite.java

```diff
@@ -510,38 +504,51 @@ private void drawMovementCost(MoveStep step, boolean isLastStep,
         graph.drawString(costString, costX, stepPos.y + 39);
         graph.setColor(col);
         graph.drawString(costString, costX - 1, stepPos.y + 38);
+    }
+
+    private void drawTMMAndRolls(MoveStep step, boolean isLastStep, boolean jumped, Game game,
```

Comment:
Apparently, the parameter "isLastStep" is unused. Let's figure out what's up with that.